aws/logs_monitoring/lambda_function.py

```diff
@@ -999,7 +1019,7 @@ def cwevent_handler(event, metadata):
 def sns_handler(event, metadata):
     data = event
     # Set the source on the log
-    metadata[DD_SOURCE] = parse_event_source(event, "sns")
+    metadata[DD_SOURCE] = "sns"
```

Comment:
No need to pass the event to parse_event_source since we already know this is SNS